src/phpDebug.ts

```diff
@@ -311,6 +317,12 @@ class PhpDebugSession extends vscode.DebugSession {
                     this.sendErrorResponse(response, <Error>error)
                 })
                 server.listen(args.port || 9000, (error: NodeJS.ErrnoException) => (error ? reject(error) : resolve()))
+                if (args.openUrl) {
+                    let proc = open(appendQuery(args.openUrl, 'XDEBUG_SESSION_START=vscode'))
+                    proc.stderr.on('data', (data: any) => {
```

Comment:
`data` is not `any`, it's either a string or a Buffer, in which case the logged output would be useless (octets). I don't know from the top of my head what it is, but it should not be typed `any`.